Main_Project/src/main/java/com/italent/controller/ItalentUsersAndRoles.java

```diff
@@ -0,0 +1,58 @@
+package com.italent.controller;
+
+import java.io.FileInputStream;
+import java.util.Properties;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.italent.dao.LithiumMessageDAO;
+import com.italent.model.Users;
+import com.italent.utils.LithiumUtils;
+
+public class ItalentUsersAndRoles {
+
+    public static final Logger logger = LogManager.getLogger(ItalentUsersAndRoles.class.getName());
+
+    public static void main(String[] args) throws Exception {
+        Properties prop = new Properties();
+        prop.load(new FileInputStream("./config/application.props"));
+        String accTokenUrl = prop.getProperty("GET_SESSION_KEY_URL");
+        logger.info("Lithium session key api : " + accTokenUrl);
+        LithiumUtils lithiumUtils = new LithiumUtils();
+        String accToken = ItalentDemo.generateSessionKey(accTokenUrl, lithiumUtils);
+        gettingUserList(accToken);
+    }
+
+    private static void gettingUserList(String sessionKey) throws Exception {
+        LithiumUtils lithiumUtils = new LithiumUtils();
+        Properties prop = new Properties();
```

Comment:
already create object for propeties in main class. why again create in this class. to resolve this add global variable in class and create object in main method .